vcpkg.json

```diff
@@ -0,0 +1,171 @@
+{
```

Comment:
Can we have a dedicated folder for such configs?

Reply:
no, the idea that they work out of box because the are located in root of  source tree.
When you run certain tool (vcpkg, conan or  IDE which support it), it finds config in the root